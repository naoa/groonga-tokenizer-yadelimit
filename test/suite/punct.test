register tokenizers/yadelimit

tokenize TokenYaDelimitPunct "a iu. eo, kaki@kuke.com"
