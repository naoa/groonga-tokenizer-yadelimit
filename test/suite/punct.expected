register tokenizers/yadelimit
[[0,0.0,0.0],true]
tokenize TokenYaDelimitPunct "a iu. eo, kaki@kuke.com"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "a iu",
      "position": 0
    },
    {
      "value": " eo",
      "position": 1
    },
    {
      "value": " kaki",
      "position": 2
    },
    {
      "value": "kuke",
      "position": 3
    },
    {
      "value": "com",
      "position": 4
    }
  ]
]
